### ISSUE TITLE
Visual design / Icons - A11y corrections

### ISSUE THREAD

**Issue by yesenia-gtz** [Office-of-Digital-Services/California-State-Web-Template-Website#965]
**Missing button in form**
The page contains a form but there is no submit button.
- [ ] The form should be provided with a submit button to maximize predictability.
- [ ] Use one of the following techniques from [WCAG 2](https://www.w3.org/TR/WCAG21/)
1. [ARIA4](https://www.w3.org/TR/WCAG20-TECHS/ARIA4) Using a WAI-ARIA role to expose the role of a user interface component
2. [H32](https://www.w3.org/TR/WCAG20-TECHS/H32) Providing submit buttons
![image (2)](https://user-images.githubusercontent.com/98193284/207433692-ace8a93d-7d47-408c-89c8-a4cebd4a1778.png)

**Text visibility**
Bump up text-sizes to 2pt on the icons library.
- [ ] Text can be resized without assistive technology up to the user agent maximum without loss of content and functionality. Standard font size should be no lower than 12pt.
![image](https://user-images.githubusercontent.com/98193284/207433800-89acd41b-22ea-4c95-b62f-515fb11b8595.png)

**Comment by yesenia-gtz** [comment]
@pglomboy this is ready for your review

**Comment by pglomboy** [comment]
@Greg-Watanabe can I get the link for the icons page please

**Comment by yesenia-gtz** [comment]
@pglomboy  https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/visual-design/icons.html#Search-icons

**Comment by pglomboy** [comment]
@Greg-Watanabe can we bump up the font size by 1pt please: from font-size-13 m-0 to font-size-14 m-0

**Comment by pglomboy** [comment]
@Greg-Watanabe button still needs a text label: 
Recommendation - aria-label="Close"

**Comment by Greg-Watanabe** [comment]
@pglomboy this is done
https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/visual-design/icons.html

**Comment by yesenia-gtz** [comment]
@pglomboy ready for your review

**Comment by pglomboy** [comment]
@Greg-Watanabe @sharon-rabang @hamiltonryan for better visibility, can we bump up the On this page text to 1.1em and the submit button of the browse icon still needs to be updated. Otherwise, other a11y issues are resolved.
![image](https://user-images.githubusercontent.com/120594656/209005673-561938a2-68d8-4a26-aee8-88eb4a1b3d0e.png)

**Comment by yesenia-gtz** [comment]
@sharon-rabang or @extinah please advise on questions above

**Comment by extinah** [comment]
@pglomboy we'll resolve this in a page navigation issue (i'll update the other issue). We're working with greg on this one.

**Comment by extinah** [comment]
@pglomboy please work with Ryan on the update to the icon (search, X, etc). @hamiltonryan is already working on this. Thanks.

**Comment by hamiltonryan** [comment]
I showed Pia the new design with the actual search button, and she confirmed that should work for the screen readers. The new design is updated in figma. 

[Figma Design](https://www.figma.com/file/nVVip6nhCxHQnzPUnVf1bb/V6--State-Web-Template-official?node-id=1679%3A17045&t=mPg5eAYF1WfyQrGx-4)

We are updating the specs on it too, but here is the quick version:

- The button (now the search icon) will submit the search. 
- Once the user starts typing, an "x" icon will appear to the left of the search icon
- If the "x" icon is clicked it clears the field 

@Greg-Watanabe @extinah @sharon-rabang @yesenia-gtz

**Comment by Greg-Watanabe** [comment]
@hamiltonryan @pglomboy 
Re: submit button
I already fixed this by adding a sr-only submit button, so can we leave this as is?

**Comment by hamiltonryan** [comment]
@Greg-Watanabe 
We would like it to be updated to the new version, for overall usability, if that would be possible?

@pglomboy @sharon-rabang @extinah

**Comment by Greg-Watanabe** [comment]
@hamiltonryan @pglomboy @sharon-rabang @extinah 
This is done
https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/visual-design/icons.html

**Comment by sharon-rabang** [comment]
@Greg-Watanabe It looks great, Thanks!
Feedback: Since the search (magnifying glass) icon's hover says "submit", can the reset "x" icon hover say "clear"?

**Comment by extinah** [comment]
This does look great. Thank you so much, Greg.

**Comment by sharon-rabang** [comment]
> 

Not sure if this result is related: 
![image](https://user-images.githubusercontent.com/99768807/209187940-d90df8e1-413c-4170-973b-f6b920647cbb.png)

**Comment by hamiltonryan** [comment]
I tried in edge and chrome and couldn't get that error, but I do get that drop down with past results too.

**Comment by yesenia-gtz** [comment]
In that case, I will close it out. thanks :)

**Comment by sharon-rabang** [comment]
> @Greg-Watanabe It looks great, Thanks! Feedback: Since the search (magnifying glass) icon's hover says "submit", can the reset "x" icon hover say "clear"?

@yesenia-gtz FYI - This is still a design fix for Greg to look into.

**Comment by yesenia-gtz** [comment]
@Greg-Watanabe let us know when the following feedback is completed.

Feedback: Since the search (magnifying glass) icon's hover says "submit", can the reset "x" icon hover say "clear"?

**Comment by Greg-Watanabe** [comment]
@yesenia-gtz @sharon-rabang 
This is done
https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/visual-design/icons.html

**Comment by extinah** [comment]
This looks great. Thanks so much, Greg.
